CompareTwo strings.py

```diff
@@ -0,0 +1,20 @@
+a = (input("a is "))
+b = (input("b is "))
+
+
+def sum(input):
+ for j in input:
+  x = 0
+  x = int(j) + int(x)
```

Comment:
int(j) will throw exception
int(x) - implicitly converting an int to integer again - just put x
x=x+1
trim the inputs , else trailing spaces will be considered